src/main/java/tahafurkan/sandbox/familybudgetapp/controller/SpendingController.java

```diff
@@ -15,11 +15,18 @@ public class SpendingController {
     SpendingService spendingService;
 
     @GetMapping(value = "/spendings")
-    public List<Spending> getAll(){
+    public List<Spending> getAll() {
         return spendingService.getAll();
     }
+
+
     @PostMapping(value = "/spendings")
-    public Spending create(@RequestBody Spending spending){
+    public Spending create(@RequestBody Spending spending) {
         return spendingService.create(spending);
     }
+
+    @GetMapping(value = "/spendings/{id}")
+    public List<Spending> getByUserId(@PathVariable Integer id) {
```

Comment:
Bu mapping dogru olmamis; `@GetMapping(value = "/spendings/{id}")` buradan anlasilan spending' id'ye gore bir kayit getirilmesi olur.
Zaten `User`'in harcamalarini donuyorsak bu `UserController`'da olmali.